src/app/portlets/content-types/fields/service/field.service.spec.ts

```diff
@@ -41,4 +42,84 @@ describe('FieldService', () => {
 
         expect(this.response).toEqual(mockResponse.entity);
     }));
-});
\ No newline at end of file
+
+    describe('Save Fields', () => {
+        beforeEach(() => {
+            this.mockData = [
+                {
+                    clazz: 'com.dotcms.contenttype.model.field.ImmutableRadioField',
+                    name: 'Hello World',
+                },
+                {
+                    clazz: 'com.dotcms.contenttype.model.field.ImmutableLineDividerField',
+                },
+            ];
+
+            this.fieldService.saveFields('1', this.mockData).subscribe(res => this.response = JSON.parse(res));
+
+            this.lastConnection.mockRespond(new Response(new ResponseOptions({
+                body: {
+                    entity: JSON.stringify(this.mockData)
+                }
+            })));
+        });
+
+        it('should save field', () => {
+            expect(this.mockData).toEqual(this.response);
+            expect(this.lastConnection.request.url).toContain('v1/contenttype/1/fields');
+            expect(2).toBe(this.lastConnection.request.method); // 2 is PUT method
```

Comment:
When you read assertions in unit tests it should read always:

expect that "OUR THING" is equal to WHATEVER like:

```expect(this.lastConnection.request.method).toBe(2)```

In just this test you are doing it both way.

Reply:
2 if the extected value